requirements.txt

```diff
@@ -2,7 +2,7 @@ python>=3.9     # sentinel1-reader requirement
 numpy           # sentinel1-reader requirement
 lxml            # sentinel1-reader requirement
 gdal>=3
-#isce3        # since the conda-installed isce3 is not the most updated version, installing isce3 from stratch is recommended, to stay in sync with isce3 development.
```

Comment:
let's get this pinned like in #170  so that we can correctly install the needed version